x/gamm/pool-models/balancer/pool_utils.go

```diff
@@ -0,0 +1,230 @@
+package balancer
```

Comment:
Stableswap package seems to keep similar utility files in `amm.go` . Let's make the names consistent between the 2 packages

Either  `pool_utils.go` or `amm.go` works IMO